python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -339,6 +361,43 @@ async def setup_laser(
         await self.change_laser_optical_configuration(optical_configuration)
         await self.change_laser_wavelength(wavelength, use_projector)
 
+    async def setup_cbp(
+        self,
+        azimuth: float,
+        elevation: float,
+        mask: int,
+        focus: float,
+        rotation: float,
+    ) -> None:
+        """Perform all steps for preparing the CBP for measurements.
+
+        Parameters
+        ----------
+        az : `float`
+            Azimuth of CBP in degrees
+        el : `float`
+            Elevation of CBP in degrees
+        mask : `int`
+            Mask number to use
+        focus: `float`
+            Focus position in um
+        rot: `int`
+            Rotator position of mask in degrees
+            Default 0
+        """
+        timeout = 60
```

Comment:
Please, don't use local timeout variable, we define some [default timeouts](https://github.com/lsst-ts/ts_observatory_control/blob/09ee0a6aac516108684cc77ba0d8630c6c0a3d56/python/lsst/ts/observatory/control/remote_group.py#L260-L262) in the base class. Chose one of those or make a custom one at the class level.